persistence/src/commonMain/db_user/com/wire/kalium/persistence/Users.sq

```diff
@@ -40,7 +40,7 @@ VALUES(?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)
 ON CONFLICT(qualified_id) DO UPDATE SET
 name = excluded.name,
 handle = excluded.handle,
-email = excluded.email,
+email = coalesce(excluded.email, email),
```

Comment:
This isn't necessary now that you exclude self when syncing users profiles? Also if the self user removes his email for some reason this would prevent from being removed? Intentional?

Reply:
i was experimenting with the reason and forgot to revert some changes.